### ISSUE TITLE
Autoplay calls get command twice

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-app#1545]
## The Issue
If autoplay is enabled and you visit a media claim where the functionality is triggered, the get command is called twice (can see this in network logs and in daemon - says Already have a file for xxxxx).  This is fairly transparent to the user, but it should only be called once. 

## System Configuration

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

**Comment by tzarebczan** [comment]
Sometimes this happens to me even without autoplay on, but can't reproduce consistently.

**Comment by tzarebczan** [comment]
This also results in 2 file view analytics API calls. We should get this fixed ASAP.

**Comment by tiger5226** [comment]
solved with https://github.com/lbryio/lbry-app/pull/1622

This was happening because when the auto play was implemented a few months ago a direct call to load a video was added. However, this was not needed because a simple play call would already by default load a video via the same call already. So what ended up happening (if the user did not already have the content) is that it would get called twice for no reason. 

@seanyesmunt I am not sure I understand what the if else if for exactly, but I did not want to mess with that, for fear of breaking something else.
